client/rpc-core/txpool/Cargo.toml

```diff
@@ -16,4 +16,4 @@ jsonrpc-derive = "18.0.0"
 serde = { version = "1.0", features = [ "derive" ] }
 serde_json = "1.0"
 
-fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "moonbeam-polkadot-v0.9.13" }
+fc-rpc-core = { git = "https://github.com/purestake/frontier", branch = "mb-v0.9.13-elois-client-v0.19" }
```

Comment:
Why use a different branch ? It should stay the same no ?

Reply:
This will be the case when this PR will be merged: https://github.com/PureStake/frontier/pull/31

Reply:
Makes sense